apistar/codecs/jsonschema.py

```diff
@@ -100,29 +105,40 @@ def load_type(typename, struct, allow_null):
     if typename == 'string':
         if 'minLength' in struct:
             attrs['min_length'] = struct['minLength']
+
         if 'maxLength' in struct:
             attrs['max_length'] = struct['maxLength']
+
         if 'pattern' in struct:
             attrs['pattern'] = struct['pattern']
+
         if 'format' in struct:
             attrs['format'] = struct['format']
+
         return validators.String(**attrs)
 
     if typename in ['number', 'integer']:
         if 'minimum' in struct:
             attrs['minimum'] = struct['minimum']
+
         if 'maximum' in struct:
             attrs['maximum'] = struct['maximum']
+
         if 'exclusiveMinimum' in struct:
             attrs['exclusive_minimum'] = struct['exclusiveMinimum']
+
         if 'exclusiveMaximum' in struct:
             attrs['exclusive_maximum'] = struct['exclusiveMaximum']
+
         if 'multipleOf' in struct:
             attrs['multiple_of'] = struct['multipleOf']
+
         if 'format' in struct:
             attrs['format'] = struct['format']
+
         if typename == 'integer':
             return validators.Integer(**attrs)
+
```

Comment:
Would need to review the spacing changes more thoroughly to come to a judgement.